### ISSUE TITLE
Strength of Earth/Stoneskin Totems no longer providing buffs

### ISSUE THREAD

**Issue by QueenOtter** [slowtorta/turtlewow-bug-tracker#3512]
**Please, provide a clear description what the bug is:**
No longer getting buffs from my own Totems such as Strength of Earth and Stoneskin. They cease to provide their buffs within a 15 minute timeframe for an undiscernable reason. Thought it might be related to new zones, but perhaps not.

**Steps to reproduce the behavior with as much detail as possible:**

1. Log in.
2. Party up in The Dawning Isles
3. Set totem down, receive buff in the Dawning Isles
4. Have ~15 minutes elapse in play time in The Dawning Isles
5. Set totem down, no longer receive buff in The Dawning Isles

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

NPC: https://database.turtle-wow.org/?npc=
Item: https://database.turtle-wow.org/?item=
Quest: https://database.turtle-wow.org/?quest=
Spell: https://database.turtle-wow.org/?spell=8076, https://database.turtle-wow.org/?spell=8072
Object: https://database.turtle-wow.org/?object=

**Expected behavior. Describe how it should work:**
I expect to have my totems provide me their appropriate buffs when I place them down, not have them as mana-cost decorations.

![image](https://github.com/slowtorta/turtlewow-bug-tracker/assets/76966656/4794428d-db16-44b2-860c-58c2f74cd3a6)
![image (2)](https://github.com/slowtorta/turtlewow-bug-tracker/assets/76966656/bbbea012-cc5a-429b-a6c3-343aa8b4a2b5)
![image (1)](https://github.com/slowtorta/turtlewow-bug-tracker/assets/76966656/d42d26e4-66bc-4261-bb68-1e751d4fb077)

**Comment by QueenOtter** [comment]
After further testing, it seems to stem from the party system. I have tested doing great distances from Undercity to Sepulcher and back first solo and then group. My buff would apply properly each time I deployed the totem upon landing.

Solo Test
4 times flew between Undercity and Sepulcher. Totem buff applied properly

Group Tests
Flew multiple times between Undercity and Sepulcher, retain buff for around 3 flights before it ceases to apply it. Group tests after solo tests result in me no longer gaining the buff if partied in Undercity and the totem is recast. Party does not benefit from totem buff as a whole.

**Comment by Baithoven56** [comment]
I've also run into this issue. 

In a party of 3 -- 2 of us(me included) being flagged for PvP -- my totem buffs would not apply. After leaving the group they started to work. I then rejoined the group and kicked the one that was not flagged for PvP and the buffs would still not work. After that I reinvited the non-flagged friend. This was when I was in Ratchet, and the other 2 in Durotar.

I then while in this party ran to the non-flagged friend in Durotar and threw down my totems. He received the buffs but I did not. After that, I ran to my friend that was flagged in Sen'jin and threw the totems down again. This time each of them worked on both of us. 

After that test, I hearthed back to Razor Hill while still in the party and threw the totems down and the buffs worked still. But then halfway on my run up to Orgrimmar, in the party, but solo, I tried to throw down my totems, and received no buffs. 

I have yet to test in instances, and I strongly hope it's a non-issue there. I can deal with not being able to be in a party with friends, but it would be a shame if we also couldn't run dungeons together and all of us benefit from the totem buffs.
